frontend/src/components/Game.css

```diff
@@ -0,0 +1,75 @@
+.buttonDir {
+  background-color: rgb(0, 0, 0);
+  color: white;
+  border: none;
```

Comment:
Maybe underline or some other effect? Currently, they don't look clickable to me and it may be confusing for someone playing the game for the first time